services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/modelsTable.tsx

```diff
@@ -15,6 +19,37 @@ const ModelsTable = ({ activityId, prompt }) => {
     queryFn: fetchModels
   });
 
+  const updateModelNote = (id, notes)  => {
+    request.put({
+      url: `${process.env.DEFAULT_URL}/api/v1/comprehension/automl_models/${id}`,
+      json: { automl_model: {notes} }, },
+      (error, httpStatus, body) => {
+        if (error) {
+        } else if (httpStatus.statusCode === 204 || httpStatus.statusCode === 200) {
+        }
+      });
+  }
+
+  const NotesWrapper = (props) => {
+    const { modelId, initialNotes } = props;
+    const [noteText, setNoteText] = React.useState(initialNotes);
+    
+    const handleNoteChange = (e, modelId) => {
+      setNoteText(e.target.value)
+      updateModelNote(modelId, e.target.value )
```

Comment:
We have an opportunity to use useDebounce() here so that not every character is PUT'd to the API. Thoughts?

Reply:
so correct me if I'm wrong... but looking at the notion doc for this, I'm not sure we have this in the right place. I think we just want to display the note data here, but edit it on the view page: https://www.quill.org/cms/comprehension#/activities/87/semantic-labels/model/8

If you look here at the [Model component](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/model.tsx) which is rendered for the view, you can see that there is already code for this commented out that will satisfy the spec details.

As for adding model notes on creation, in the [ModelForm component](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Staff/components/comprehension/semanticRules/modelForm.tsx), we would want this `TextArea` component there with the placeholder text specified in the spec.

Reply:
> When you pull up the page, you can write directly into the notes (it is a textbox in the editable state). 

I'm pretty sure we want to model notes editable inline. See the above quote and also the mockup in the notion doc.

You're right, I need to add the default text. I'm going to add that after, since I don't want the PR to be held up until I get back on Monday.

Reply:
FYI, more discussion in the comments on inline vs. not inline. I'm going with inline for now, since that's what I've already written: https://www.notion.so/quill/Semantic-Labels-Page-Add-Model-Notes-1cf13134be354203a170d7eab0ad8c9e